lib/carbon/database.py

```diff
@@ -77,6 +77,13 @@ def __init__(self, settings):
         else:
           log.err("WHISPER_LOCK_WRITES is enabled but import of fcntl module failed.")
 
+      if settings.WHISPER_FADVISE_RANDOM:
+        if whisper.CAN_FADVISE:
```

Comment:
Maybe we need to add try-catch block here for ignoring of non-compatible whisper lib?